### ISSUE TITLE
Search results button text doesn't have enough contrast

### ISSUE THREAD

**Issue by thanyabegum** [pulibrary/dpul#1218]
The buttons for refining search results (e.g. sort and number per page) don't have enough contrast between the text of the button and the background. One solution would be to make the text darker, while another would be to make the background lighter.

<img width="531" alt="Screen Shot 2021-11-05 at 2 47 47 PM" src="https://user-images.githubusercontent.com/66143640/140563532-e6fb07bf-c062-403b-8645-392682fe7e00.png">

[Learn more about this failing Lighthouse test.](https://web.dev/color-contrast/?utm_source=lighthouse&utm_medium=devtools)
